app/components/preprint-form-authors.js

```diff
@@ -236,16 +236,17 @@ export default CpPanelBodyComponent.extend(Analytics, {
                 draggedContrib.rollbackAttributes();
             });
         },
-        // Action used by the pagination-pager compoent to the handle user-click event.
+        // Action used by the pagination-pager component to the handle user-click event.
         pageChanged(current) {
             let query = this.get('query');
             if (query) {
                 this.attrs.findContributors(query, current).then(() => {
                     this.set('addState', 'searchView');
-                    this.set('currentPage', current);
                 }, () => {
                     this.get('toast').error('Could not perform search query.');
                     this.highlightSuccessOrFailure('author-search-box', this, 'error');
+                }).then(() => {
+                   this.set('currentPage', current);
```

Comment:
It's not necessary to use a separate `.then`. But if it recommended to use a separate `.then` vs `.catch`. The specific line affected by this is Line 245.

A general recommendation of good habit is to rewrite statements like: 
`findContributors().then( success, failure )`

to
`findContributors().then( success).catch(failure )`